package/src/index.js

```diff
@@ -17,11 +17,10 @@ export { default as LogoutButton } from "./components/LogoutButton";
  */
 import "./themes/default.css";
 
-// TODO #9: add link to upgrade guide
 Userfront.build = (toolId) => {
   console.error(`Userfront.build(${toolId}) has been removed. Instead, import the component directly:
     import { SignupForm, LoginForm, PasswordResetForm, SetNewPasswordForm, LogoutButton } from "@userfront/react"
```

Comment:
Is the `@userfront/react` portion of this incorrect?

Reply:
Good catch!